src/components/idcard/IdCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import style from './IdCard.module.css';
+
+function IdCard(props) {
+    return (
+        <div className={style.card__id}>
+            <img src={props.user.picture} alt="user"></img>
+            <div>
+                <p><strong>First name:</strong> {props.user.firstName}</p>
+                <p><strong>Last name:</strong> {props.user.lastName}</p>
+                <p><strong>Gender:</strong> {props.user.gender}</p>
+                <p><strong>Height:</strong> {props.user.height}</p>
+                <p><strong>Birth:</strong> {props.user.birth}</p>
+            </div>
+        </div>
+    )
+}
```

Comment:
Everything here looks great, although, why didn't you use directly the:

```
<IdCard
  lastName='Doe'
  firstName='John'
  gender='male'
  height={178}
  birth={new Date("1992-07-14").toLocaleDateString(undefined, dateOption)}
  picture="https://randomuser.me/api/portraits/men/44.jpg"
/>

<IdCard
  lastName='Delores '
  firstName='Obrien'
  gender='female'
  height={172}
  birth={new Date("1988-05-11").toLocaleDateString(undefined, dateOption)}
  picture="https://randomuser.me/api/portraits/women/44.jpg"
/>
```

Insted of creating a user in a App Component? To make the date look better you could just add, as you did in the user (.toLocaleDateString(undefined, dateOption)) to the tag <IdCard/>